test/org/sagebionetworks/bridge/play/controllers/StudyControllerTest.java

```diff
@@ -198,6 +208,31 @@ public void deactivateStudyThrowsGoodException() throws Exception {
     }
 
     @Test
+    public void canCreateSynapseAccount() throws Exception {
+        doReturn(mockSession).when(controller).getAuthenticatedSession(DEVELOPER);
+        TestUtils.mockPlayContextWithJson(TEST_SYNAPSE_ACCOUNT_JSON);
+        when(mockStudyService.createSynapseAccount(any())).thenReturn(TEST_SYNAPSE_USER_ID);
+
+        controller.createSynapseAccount();
+
+        Result result = controller.createSynapseAccount();
```

Comment:
You call createSynapseAccount() twice in your test.

Also, you need to verify the value that was passed into mockStudyService.